packages/devkit/src/modal.js

```diff
@@ -19,6 +19,9 @@ import minsIcon from "../styles/icons/general/mins_icon.svg"; //eslint-disable-l
 import minsHoverIcon from "../styles/icons/hover/mins_icon_h.svg"; //eslint-disable-line
 import maxIcon from "../styles/icons/general/max_icon.svg"; //eslint-disable-line
 import maxHoverIcon from "../styles/icons/hover/max_icon_h.svg"; //eslint-disable-line
+const FocusProtection = focusProtection();
```

Comment:
Las variables nunca se nombran empezando con mayúscula, aquí puedes hacer varias cosas:

```suggestion
const  { protect, unprotect }= focusProtection();
```

o

```
const  { protect: captureFocus, unprotect: releaseFocus }= focusProtection(); // o como quieras llamarlas
```

o simplemente renombrando la variable para que no parezca que es una clase con métodos estáticos:

```
const  focus = focusProtection(); // o como quieras llamarlas

...

focus.protect();

...

focus.unprotect();

```

Reply:
me gusta el aproche del telemeter `{ protect, unprotect } `segun haga falta